spec/controllers/medicaid/contact_other_address_controller_spec.rb

```diff
@@ -0,0 +1,39 @@
+require "rails_helper"
+
+RSpec.describe Medicaid::ContactOtherAddressController, type: :controller do
```

Comment:
I feel like this controller is poorly named, we should consider renaming in the future